doc/ref_arch/openstack/chapters/chapter01.rst

```diff
@@ -147,10 +147,51 @@ to address them (for example, resources autoscaling).
 Terminology
 -----------
 
-General terminology definitions can be found in
-:doc:`cntt:common/glossary` and specific terms relating
-to this reference architecture are to be found in `OpenStack Related
-Terminology :ref:`cntt:common/glossary:openstack related terminology`.
+.. glossary::
+
+    Cloud Infrastructure
+      A generic term covering **NFVI**, **IaaS** and **CaaS** capabilities -
+      essentially the infrastructure on which a **Workload** can be executed.
+
+.. note::
+
+  `The official OpenStack Glossary <https://docs.openstack.org/image-guide/common/glossary.html>`__
+  is an extensive list of OpenStack-related concepts. Some additional terms
+  used in the Reference Architecture RA-1 or used to relate RA-1 terms with
+  terms defined elsewhere.
+
+
+.. glossary::
+
```

Comment:
@collivier the GSMA uses the word "Terminology" rather than "Glossary".
During the RA1 meeting of August 1st, we agreed on the idea to have a Terminology section in order to define the terms we are using through the document and a separated Abbreviation section for acronyms.
In doing so, we will not need to use the LaTex Glossary function and also avoid the introduction of the keywords :term: to reference all the terms in the text.